responder/api.py

```diff
@@ -120,6 +122,7 @@ def __init__(
                 license=license,
                 openapi_route=openapi_route,
                 static_route=static_route,
+                api_theme=api_theme,
```

Comment:
For clarity, this variable should called `openapi_theme`

Reply:
Thanks for the guidance. I used your https://github.com/kennethreitz/responder/commit/0874ce87e98b4f75ba6bcff157c3b437414bd1fa.patch, where it was like `api_theme=api_theme`.

Now, 1107f5d039efdd2f65 adjusts the code according to your suggestions. Is it correct?